Software/dist_windows/script32.iss

```diff
@@ -57,18 +57,13 @@ Source: "content/Qt5Network.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "content/Qt5SerialPort.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "content/Qt5Widgets.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "gpl-3.0.txt"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icudt54.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icuin54.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icuuc54.dll"; DestDir: "{app}"; Flags: ignoreversion
```

Comment:
I can force it to copy it I think, by adding a command line argument. But even according to Dependency Walker, Qt5Core doesn't seem to have a hard dependency on it.

![naamloos](https://cloud.githubusercontent.com/assets/1467127/8908908/eb8f2138-347e-11e5-8bd1-624880c8f03a.png)